spec/features/manage_cmr/subscriptions/edit_subscriptions_spec.rb

```diff
@@ -0,0 +1,98 @@
+describe 'Edit/Updating Subscriptions' do
+  before do
+    login
+    allow_any_instance_of(SubscriptionPolicy).to receive(:create?).and_return(true)
+    allow_any_instance_of(SubscriptionPolicy).to receive(:show?).and_return(true)
+    allow_any_instance_of(SubscriptionPolicy).to receive(:destroy?).and_return(true)
+    allow_any_instance_of(SubscriptionPolicy).to receive(:update?).and_return(true)
+    # make a record
+    @native_id = 'test_native_id'
+    @ingest_response, @subscription = publish_new_subscription(native_id: @native_id)
+  end
+
+  context 'when visiting the show page and clicking the edit button' do
+    before do
+      VCR.use_cassette('urs/rarxd5taqea', record: :none) do
+        visit subscription_path(@ingest_response['concept_id'], subscription: @subscription, native_id: @native_id)
+        click_on 'Edit'
+      end
+    end
+
+    it 'takes the user to the edit page' do
+      expect(page).to have_content('Edit MMT_2 Subscription')
+      expect(page).to have_field('Subscription Name', with: @subscription['Name'])
+      expect(page).to have_field('Query', with: @subscription['Query'])
+      expect(page).to have_field('Subscriber', with: @subscription['SubscriberId'])
+      expect(page).to have_field('Collection Concept ID', with: @subscription['CollectionConceptId'])
+    end
+  end
+
+  context 'when visiting the edit page' do
+    before do
+      # go to show page
+      VCR.use_cassette('urs/rarxd5taqea', record: :none) do
+        visit edit_subscription_path(@ingest_response['concept_id'], subscription: @subscription, native_id: @native_id)
+      end
+    end
+
+    it 'has the right fields' do
```

Comment:
this seems like a duplicate of what's being tested on line 21, not sure it is necessary